app/src/renderer/vuex/getters.js

```diff
@@ -78,7 +80,7 @@ export const pool = state => state.pool
 export const stakingParameters = state => state.stakingParameters
 export const bondDenom = getters =>
   getters.stakingParameters.parameters &&
-  getters.stakingParameters.parameters.bond_denom || `uatom`
+  num.viewDenom(getters.stakingParameters.parameters.bond_denom || `uatom`)
```

Comment:
```suggestion
  num.viewDenom(getters.stakingParameters.parameters.bond_denom || `ATOMs`)
```

Reply:
I think this is wrong here. `bond_denom` in the happy case is `uatom`. No plural, no upper case. If we want to change this, the processing should happen inside `viewDenom`.